src/components/Books.js

```diff
@@ -1,24 +1,19 @@
+import { useSelector } from 'react-redux';
 import Book from './Book';
 
 const Books = () => {
-  const books = [
-    {
-      title: 'Harry Potter',
-      author: 'J.K. Rowling',
-    },
-    {
-      title: 'Ensayo sobre la ceguera',
-      author: 'José Saramago',
-    },
-    {
-      title: 'Dracula',
-      author: 'Bram Stocker',
-    },
-  ];
+  const books = useSelector((state) => state.books);
   return (
-    <ul>
-      {books.map((el) => <Book key={el.title} title={el.title} author={el.author} />)}
-    </ul>
+    <div>
+      {books.map((el) => (
+        <Book
+          key={el.title}
+          title={el.title}
+          author={el.author}
+          id={el.item_id}
+        />
+      ))}
```

Comment:
- [OPTIONAL] the `key` prop for each _**book**_ ought to be unique, it would be better to use the actual ids as the key instead of the title. If I have an entry of the same titles, React will warn us of such. I would advise you to make the `ids` the `key` for each book. 👍 💻 See the screenshot attached below.

![image](https://user-images.githubusercontent.com/77106962/229645715-efe0bb04-c0ea-4388-82b4-056cc9ef853f.png)